src/config/KialiIcon.tsx

```diff
@@ -37,20 +44,27 @@ export const KialiIcon: { [name: string]: React.FunctionComponent<IconProps> } =
   AngleDoubleLeft: (props: IconProps) => <AngleDoubleLeftIcon className={props.className} />,
   AngleDoubleRight: (props: IconProps) => <AngleDoubleRightIcon className={props.className} />,
   AngleDoubleUp: (props: IconProps) => <AngleDoubleUpIcon className={props.className} />,
+  AngleLeft: (props: IconProps) => <AngleLeftIcon className={props.className} />,
+  AngleRight: (props: IconProps) => <AngleRightIcon className={props.className} />,
+  AngleDown: (props: IconProps) => <AngleDownIcon className={props.className} />,
   Applications: (props: IconProps) => <ApplicationsIcon className={props.className} />,
   Bell: (props: IconProps) => <BellIcon className={props.className} />,
   CircuitBreaker: (props: IconProps) => <BoltIcon className={props.className} />,
+  Close: (props: IconProps) => <CloseIcon className={props.className} />,
   Error: (props: IconProps) => <ErrorCircleOIcon className={props.className} color={PfColors.Danger} />,
+  Help: (props: IconProps) => <HelpIcon className={props.className} />,
   Info: (props: IconProps) => <InfoAltIcon className={props.className} color={PfColors.Info} />,
-  Ok: (props: IconProps) => <OkIcon className={props.className} color={PfColors.Success} />,
   MissingSidecar: (props: IconProps) => <CodeBranchIcon className={props.className} />,
```

Comment:
Oh! I see why the VirtualService is wrong.
Apparently, the MissingSidecar and the VirtualService icons are switched.

Reply:
Ah right!

Reply:
Good catch!